contract-addresses/dependencies.ts

```diff
@@ -1,5 +1,735 @@
+    contractDependencies: ['core']
+  },
+  tribe: {
+    contractDependencies: [
+      'core',
+      'rariPool8Tribe',
+      'feiDAO',
+      'tribeRariDAO',
+      'erc20Dripper',
+      'aaveTribeIncentivesController',
+      'tribeMinter',
+      'tribeReserveStabilizer'
+    ]
+  },
+  tribeMinter: {
+    contractDependencies: ['core', 'tribeReserveStabilizer', 'tribe', 'erc20Dripper', 'feiDAOTimelock' /* Owner */]
+  },
+  feiDAO: {
+    contractDependencies: ['feiDAOTimelock', 'tribe']
+  },
+  feiDAOTimelock: {
+    contractDependencies: [
+      'core',
+      'feiDAO',
+      'fei',
+      'proxyAdmin',
+      'creamDepositWrapper',
+      'aaveTribeIncentivesController',
+      'tribeMinter',
+      'pcvGuardian'
+    ]
+  },
+  guardian: {
+    contractDependencies: ['core', 'collateralizationOracleGuardian', 'pcvGuardian'] // TODO do we want to document everything the guardian can affect. I think this should only reflect guardian-exclusive actions
+  },
+  optimisticMultisig: {
+    contractDependencies: ['optimisticTimelock']
+  },
+  optimisticTimelock: {
+    contractDependencies: [
+      'core',
+      'rewardsDistributorAdmin',
+      'tribalChiefSync',
+      'rariPool8Comptroller',
+      'optimisticMultisig',
+      'optimisticMinter',
+      'tribalChief',
+      'collateralizationOracle',
+      'collateralizationOracleWrapper',
+      'staticPcvDepositWrapper2'
+    ]
+  },
+  rariTimelock: {
+    contractDependencies: ['tribeRariDAO']
+  },
+  tribeRariDAO: {
+    contractDependencies: ['rariTimelock', 'tribe']
+  },
+  aaveEthPCVDripController: {
+    contractDependencies: ['core', 'fei', 'aaveEthPCVDeposit', 'ethReserveStabilizer']
+  },
+  bondingCurve: {
+    contractDependencies: ['core', 'fei', 'aaveEthPCVDeposit', 'compoundEthPCVDeposit', 'chainlinkEthUsdOracleWrapper']
+  },
+  compoundEthPCVDripController: {
+    contractDependencies: ['core', 'fei', 'compoundEthPCVDeposit', 'ethReserveStabilizer']
+  },
+  daiBondingCurve: {
+    contractDependencies: ['core', 'fei', 'compoundDaiPCVDeposit', 'chainlinkDaiUsdOracleWrapper']
+  },
+  daiPCVDripController: {
+    contractDependencies: ['core', 'fei', 'daiPSM', 'compoundDaiPCVDeposit']
+  },
+  daiPSM: {
+    contractDependencies: [
+      'core',
+      'fei',
+      'compoundDaiPCVDeposit',
+      'daiPCVDripController',
+      'chainlinkDaiUsdOracleWrapper',
+      'pcvGuardian'
+    ]
+  },
+  ethReserveStabilizer: {
+    contractDependencies: [
+      'core',
+      'aaveEthPCVDripController',
+      'compoundEthPCVDripController',
+      'chainlinkEthUsdOracleWrapper',
+      'pcvGuardian'
+    ]
+  },
+  tribeReserveStabilizer: {
+    contractDependencies: ['core', 'tribeUsdCompositeOracle', 'tribeMinter', 'collateralizationOracleWrapper', 'tribe']
+  },
+  aaveEthPCVDeposit: {
+    contractDependencies: ['core', 'aaveEthPCVDripController', 'bondingCurve', 'pcvGuardian']
+  },
+  aaveFeiPCVDeposit: {
+    contractDependencies: ['core', 'fei']
```

Comment:
this should also be dependent on the aFEI token

Reply:
aFEI is external